client/my-sites/exporter/export-card/index.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import flowRight from 'lodash/flowRight';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import SpinnerButton from './spinner-button';
+import FoldableCard from 'components/foldable-card';
+import analytics from 'lib/analytics';
+import Interval, { EVERY_SECOND } from 'lib/interval';
+import AdvancedSettings from './advanced-settings';
+import {
+	advancedSettingsFetch,
+	exportStatusFetch,
+	setPostType,
+	startExport,
+} from 'state/site-settings/exporter/actions';
+import {
+	shouldShowProgress,
+	getSelectedPostType,
+	isExporting,
+} from 'state/site-settings/exporter/selectors';
+
+class ExportCard extends Component {
+	componentWillMount() {
+		this.props.advancedSettingsFetch( this.props.siteId );
+	}
+
+	componentWillReceiveProps( newProps ) {
+		if ( newProps.siteId !== this.props.siteId ) {
+			this.props.advancedSettingsFetch( newProps.siteId );
+		}
+	}
+
+	trackExportClick = ( scope = 'all' ) => {
+		analytics.tracks.recordEvent( 'calypso_export_start_button_click', { scope } );
+	}
+
+	exportAll = () => {
+		this.trackExportClick();
+		this.props.startExport();
+	}
```

Comment:
It avoids the need for `.bind( this )` 😃 

Basically just saves us the extra step of doing `this.exportAll = this.exportAll.bind( this )` in the constructor (currently a [Stage 2 Proposal](https://github.com/jeffmo/es-class-public-fields)).

There's a [blurb on the React blog suggesting this](https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#autobinding) as a replacement for the `React.createClass` autobinding.

These were removed in 311bb3b as part of using Redux to add analytics